askup/models.py

```diff
@@ -291,6 +291,48 @@ class Domain(models.Model):
     name = models.CharField(max_length=50, unique=True)
 
 
+class QuestionQuerySet(models.query.QuerySet):
+    """
+    Overrides a question query set class.
+
+    Overriding a question query set class to have a control
+    over multiple deleting in admin panel.
```

Comment:
Please mention that this solution has no any technical relations to the Django admin but simply fix `QuerySet.delete`method. It used in admin but also will be applied to any part of you project where you will decide to use delete method for QuerySet => `Question.objects.all().delete()`

Reply:
done